prismic-model/json/collection-venues.json

```diff
@@ -6,18 +6,57 @@
         "label": "Title"
       }
     },
-    "description": {
```

Comment:
Is this going to break any collection venues that had descriptions?

Reply:
nope, this wasnt used in the first place. so i removed this.